### DISCUSSION TITLE
21-06-16 일일회고

### DISCUSSION THREAD

**Discussion by GleamingStar** [d-h-k/issue-tracker#72] [회고]
코쿼 졸업까지 목금토일월화수목금 9일!

**Comment by GleamingStar** [comment]
오늘은 사이드바의 모달 및 필터를 작업했습니다!
저희 user정보 같은 경우는 어떻게 관리할까요?
저희 5명의 정보를 객체로 만들어서 데이터베이스에 보관하는 방식일까요?
아니면 로그인API때 동적으로;; user가 생성되는 식일까요

**Comment by eeeesong** [comment]
오늘은 레이블 추가 화면 UI 및 API 연동 작업까지 완료하였습니다!

<img src=https://user-images.githubusercontent.com/72188416/122215873-b76b6200-cee6-11eb-9d73-35203acfb93c.png width=300>

아직 삭제나 수정을 구현하지 않아서 테스트 레이블이 너무 많이 쌓였네요🥲
내일은 삭제를 구현해서 이 쓸데없는 것들을 없애버리겠습니다ㅎㅎ
